share/spice/git_book_status/content.handlebars

```diff
@@ -0,0 +1,16 @@
+<h3 class="gbstatus--header text--primary">
+    {{! Check if the status is either good, minor or major }}
+    {{#GitbookStatus_indicator status "none"}}
+        The status of GitBook is <span class="gbstatus--good">good</span>.
+    {{else}}
+        {{#GitbookStatus_indicator status "minor"}}
+            GitBook status is <span class="gbstatus--minor">{{status}}</span>.
+        {{else}}
+            GitBook status is <span class="gbstatus--major">{{status}}</span>.
+        {{/GitbookStatus_indicator}}
+    {{/GitbookStatus_indicator}}
+    
+    {{#GitbookStatus_ifNotEmpty description}}
```

Comment:
If we use the `text` template, instead of `base` and use `title_content` instead of `content, we can define the `subtitle` in the JS, which will simplify the template and the CSS a bit.